cmd/bad-key-revoker/main.go

```diff
@@ -69,7 +69,7 @@ func (bkr *badKeyRevoker) selectUncheckedKey() (uncheckedBlockedKey, error) {
 	var row uncheckedBlockedKey
 	err := bkr.dbMap.SelectOne(
 		&row,
-		"SELECT keyHash, revokedBy FROM blockedKeys WHERE extantCertificatesChecked = false",
+		"SELECT keyHash, revokedBy FROM blockedKeys WHERE extantCertificatesChecked = false LIMIT 1",
```

Comment:
Can you reformat this as a multi-line query, starting each line with a keyword (`SELECT`, `FROM`, etc)?